examples/local-access/setup-external-connection.ipynb

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
<p>Is it possible to do this with the quickstart project, rather than telling people to create a new one?</p><p> </p><p>Also, is there a way to retrieve the project_id from an environment variable also? Might seem less hacky than "grab this thing from your URL" ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/121/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Could be! Maybe the pytorch quickstart? I didn't want people to get confused about all the notebooks already in the quick start is the problem- if they don't intend to use them, then that seems like it would be distracting at best.